Fromm: Luca Bruno <lucab@debian.org>
Description: Remove stage1 extra dependencies
 Fetching extra dev-dependencies is not yet supported by bootstrap.py
 and they are not need to build. However, cargo will try to download
 them before building stage1.
Forwarded: not-needed
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -17,8 +17,9 @@
 path = "src/cargo/lib.rs"
 
 [dependencies]
-advapi32-sys = "0.1"
-crates-io = { path = "src/crates-io", version = "0.1" }
+#advapi32-sys = "0.1"
+crates-io = "0.1"
+#crates-io = { path = "src/crates-io", version = "0.1" }
 crossbeam = "0.1"
 curl = "0.2"
 docopt = "0.6"
@@ -28,7 +29,7 @@
 git2 = "0.3"
 git2-curl = "0.3"
 glob = "0.2"
-kernel32-sys = "0.1"
+#kernel32-sys = "0.1"
 libc = "0.1"
 libgit2-sys = "0.3"
 log = "0.3"
@@ -41,13 +42,13 @@
 time = "0.1"
 toml = "0.1"
 url = "0.2"
-winapi = "0.2"
+#winapi = "0.2"
 
-[dev-dependencies]
-tempdir = "0.3"
-hamcrest = "0.1"
-bufstream = "0.1"
-filetime = "0.1"
+#[dev-dependencies]
+#tempdir = "0.3"
+#hamcrest = "0.1"
+#bufstream = "0.1"
+#filetime = "0.1"
 
 [[bin]]
 name = "cargo"
